CONTRIBUTING.md

```diff
@@ -184,6 +185,16 @@ If you are adding a new feature, don't forget to edit the file
 `test/runnableTestSets.csv`. This file disables tests for features we
 have not yet implemented.
 
+If you want to check the performance of fontoxpath, you can pick some of
+qt3tests up randomly and run them via benchmark.js. As default, there is
+list. You can create the list which has 50 random qt3tests by running
+`npm run performance regenerate`. If you want to create a list which has
```

Comment:
ah, for the --generate, I think you'll need to run it like `npm run performance -- --generate`. The first `--` indicate the start of the arguments for npm.

Reply:
done